treeherder/log_parser/parsers.py

```diff
@@ -380,6 +380,7 @@ class ErrorParser(ParserBase):
         r"|.*\.js: line \d+, col \d+, Error -"
         r"|^\[taskcluster\] Error:"
         r"|^\[[\w._-]+:(?:error|exception)\]"
+        r"|thread .* panicked"
```

Comment:
I feel like it might be better to write this as the slightly more specific `thread '\w+' panicked`

You should be able to add a test in `tests/log_parser/test_error_parser.py` fairly easily just by putting some log samples in there. If that's not working out let me know and I can write some stuff for you.